test/Adapter/DbTable/CredentialTreatmentAdapterTest.php

```diff
@@ -177,6 +181,10 @@ public function testGetSpecificResultRow(): void
      */
     public function testGetOmittedResultRow(): void
     {
+        if (PHP_VERSION_ID >= 80100) {
```

Comment:
Instead of skipping, can we compare the result with a new `stdClass` perhaps? (`assertEquals()`?)

Reply:
yup, works